src/components/wallet/NewMnemonicConfirm.tsx

```diff
@@ -5,6 +5,8 @@ import { Card, Col, Row, Button, Form } from 'antd'
 import { Store } from 'antd/lib/form/interface'
 import { v4 as uuidv4 } from 'uuid'
 
+import { isSelectedFactory } from '../../helpers/isSelectedHelper'
```

Comment:
How about to name this helper just `isSelected` (w/o `factory`) and the module `helpers/arrayHelper` or `helpers/listHelper`?

Reply:
but.... this is a factory for isSelected functions😅

Reply:
Ah, I've missed that. `factory` is fine in this case ;)